### PR TITLE
[Doc Improvement][App icon badging example snippet update][3520353]

### PR DESCRIPTION
**Pull request by v-sdhakshina**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/end-user-purchase-experience.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2fb7b72bc4ea0150ff72d374b6389ded7b19a947/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fend-user-purchase-experience.md) | 90 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0600f5c8-bf86-4b20-be0a-960259750fa4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/end-user-purchase-experience.md](https://github.com/MicrosoftDocs/msteams-docs/blob/958ef6895b0473c1fabf2af7fc6fb27c8a6164ee/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fend-user-purchase-experience.md) | 90 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c4b257f3-fd2e-420f-872d-aac4d1a7e9fe) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f94c17b3218fea03ce09b81117f746368e5b5d49/msteams-platform%2Fapps-in-teams-meetings%2Fapp-icon-badging-for-your-app.md) | 89 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/111d9eb8-74e8-420a-a207-8061b244c93b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5b27aa5aa1014db3bd958a46d87672fcb3ec3aff/msteams-platform%2Fapps-in-teams-meetings%2Fapp-icon-badging-for-your-app.md) | 89 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed18773e-046f-4fde-aba1-58445eb979b4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9b0606fdb0e8d962408708578930d6af63e34b84/msteams-platform%2Fapps-in-teams-meetings%2Fapp-icon-badging-for-your-app.md) | 89 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/90cc9df1-38f5-4788-96ca-a4e829765213) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b090376ee07022f01583a52af7895418ae66123d/msteams-platform%2Fapps-in-teams-meetings%2Fapp-icon-badging-for-your-app.md) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/622e1248-60c5-479a-90dc-d77dd7dd7f1b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/926ad99f02bca920069e5f698c210e936536c021/msteams-platform%2Fapps-in-teams-meetings%2Fapp-icon-badging-for-your-app.md) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7a92f337-f078-45ac-b6ed-b89e5a0c7027) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/82b4fb65377eb046f780abf0882cc545dc0fd19f/msteams-platform%2Fapps-in-teams-meetings%2Fapp-icon-badging-for-your-app.md) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c1761f93-8f15-4d02-85f0-3ebd270e86d9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]
Please add links to respective samples where this snippet was taken from.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2becfa6f81395e9adccacc8fd457c6fe96d0a20f/msteams-platform%2Fapps-in-teams-meetings%2Fapp-icon-badging-for-your-app.md) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3f90bab5-c0e5-45d7-ab5e-66d5d31ce87a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md [hunk 1]
```diff
@@ -63,7 +63,25 @@ App can also pass the `tabEntityId`, if there are multiple instances of the app
 
 ### Example
 
-The following is an example of a payload:
+The following code is an example of the payload:
+
+# [C#](#tab/csharp)
+
+[Sample code reference](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/8535a89765319f3940e2a4eac7a9eb517a980211/samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/Bots/AppIconBadgingInMeeting.cs#L108)
```
Comment (Wajeed-msft) [inline msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md:70]:
```suggestion
[Sample code reference](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/Bots/AppIconBadgingInMeeting.cs#L108)
```

File: msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md [hunk 2]
```diff
@@ -63,7 +63,25 @@ App can also pass the `tabEntityId`, if there are multiple instances of the app
...
+
+```csharp
+TeamsInfo.SendMeetingNotificationAsync(context, notificationPayload, meetingId);
+```
+
+# [JavaScript](#tab/javascript)
+
+[Sample code reference](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/8535a89765319f3940e2a4eac7a9eb517a980211/samples/meetings-app-icon-badging/nodejs/bots/teamsBot.js#L83)
```
Comment (Wajeed-msft) [inline msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md:78]:
```suggestion
[Sample code reference](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/meetings-app-icon-badging/nodejs/bots/teamsBot.js#L83)
```

**Review by Wajeed-msft — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c81a0c8481fb37385a972bbddd1e15a5b282a576/msteams-platform%2Fapps-in-teams-meetings%2Fapp-icon-badging-for-your-app.md) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ad13744b-7fbe-46c3-b65c-119c5954a031) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/311f538abf25cdd931d385b94c432771544b1db7/msteams-platform%2Fapps-in-teams-meetings%2Fapp-icon-badging-for-your-app.md) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/11945497-1f06-49bc-9ab0-c0e2eff782e4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7e39480e2f128f5d3704974086a52764fdc180b0/msteams-platform%2Fapps-in-teams-meetings%2Fapp-icon-badging-for-your-app.md) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/359b9d4e-d7d1-4437-84e3-e23f7de2c0fa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-ypalikila — DISMISSED** [review]
Approved.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2d0ceabd12231405328e847c0ff40f99a3d093b1/msteams-platform%2Fapps-in-teams-meetings%2Fapp-icon-badging-for-your-app.md) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1a23041a-33e7-4dc2-b302-e689ec2782b2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/app-icon-badging-for-your-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e71c3e7fe04270c844d620d24fc8a9d0456adf9a/msteams-platform%2Fapps-in-teams-meetings%2Fapp-icon-badging-for-your-app.md) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ade61fe8-0ea9-4808-838f-09210c33512b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**